src/exception_summary.jl

```diff
@@ -143,9 +143,10 @@ function _summarize_exception(io::IO, exc, stack; prefix = nothing)
         end
     end
     # Now print just the very first frame we've collected:
-    if isempty(bt)
+    if ! @isdefined :bt || isempty(bt)
```

Comment:
Perhaps we should define it as empty vector instead of `local bt`, in case it doesn't enter the loop.

Reply:
👍 done in bd720cf6b42dccd5c7b062a3030a4c9ca7e0812e.

Reply:
nice suggestion! FWIW Sagar, the previous code had a small typo in it. You want either of these two options: `@isdefined(bt)` or `isdefined(@__MODULE__, :bt)`. What you have is checking if the symbol exists and that will be a type error:
```julia
julia> @isdefined(:hi)
ERROR: LoadError: MethodError: no method matching var"@isdefined"(::LineNumberNode, ::Module, ::QuoteNode)
```

Reply:
😮  Thanks for pointing that out, @NHDaly! I see how this happened. Initially (before I filed this PR) I had written:

```
if !isdefined(Base, :bt) || isempty(bt)
```

which was correct (I think), but I didn't like it stylistically, so I pattern-matched changed it to:

```
if ! @isdefined :bt || isempty(bt)
```

which was definitely incorrect!